tests/phases/design/test_mooring_system_design.py

```diff
@@ -45,6 +49,86 @@ def test_rating_sweeip(rating):
     assert m.total_cost
 
 
+def test_catenary_mooring_system_kwargs():
```

Comment:
We should also test that default and setting caternary are the same because we're adding in the option to specify. I didn't add it above, but this is to ensure consistent behavior for old users.